prow/jobs/kyma/kyma-integration-gardener.yaml

```diff
@@ -240,6 +240,54 @@ presubmits: # runs on PRs
               requests:
                 memory: 1Gi
                 cpu: 400m
+    - name: pre-main-kyma-gardener-gcp-api-gateway
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-kyma-gardener-gcp-api-gateway"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-cluster-version: "true"
+        preset-debug-commando-oom: "true"
+        preset-gardener-gcp-kyma-integration: "true"
+        preset-kyma-cli-stable: "true"
+      optional: true
+      skip_report: false
+      decorate: true
+      decoration_config:
+        grace_period: 600000000000
+        run_if_changed: ^(((resources/api-gateway)\S+)(\.[^.][^.][^.]+$|\.[^.][^dD]$|\.[^mM][^.]$|\.[^.]$|/[^.]+$))
+        timeout: 14400000000000
+      path_alias: github.com/kyma-project/kyma
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
```

Comment:
i think this one is not necessary anymore

Reply:
this also added by default by the render templates tool